callback.go

```diff
@@ -0,0 +1,110 @@
+package hcsshim
+
+import (
+	"errors"
+	"syscall"
+)
+
+var (
+	hcsNotificationInvalid hcsNotification = 0x00000000
+
+	/// Notifications for HCS_SYSTEM handles
+	hcsNotificationSystemExited          hcsNotification = 0x00000001
+	hcsNotificationSystemCreateCompleted hcsNotification = 0x00000002
+	hcsNotificationSystemStartCompleted  hcsNotification = 0x00000003
+	hcsNotificationSystemPauseCompleted  hcsNotification = 0x00000004
+	hcsNotificationSystemResumeCompleted hcsNotification = 0x00000005
+
+	/// Notifications for HCS_PROCESS handles
+	hcsNotificationProcessExited hcsNotification = 0x00010000
+
+	/// Common notifications
+	hcsNotificationServiceDisconnect hcsNotification = 0x01000000
+
+	/// The upper 4 bits are reserved for flags. See HCS_NOTIFICATION_FLAGS
+	hcsNotificationFlagsReserved hcsNotification = 0xF0000000
+)
+
+var (
+	errSOK = syscall.Errno(0)
+
+	notificationWatcherCallback        = syscall.NewCallback(notificationWatcher)
+	processNotificationWatcherCallback = syscall.NewCallback(processNotificationWatcher)
+
+	ErrUnexpectedContainerExit = errors.New("Unexpected container exit")
```

Comment:
No, I mean the text itself is not supposed to start with a capital letter. It should be "unexpected container exit"